Firestore/core/src/firebase/firestore/remote/serializer.cc

```diff
@@ -95,6 +98,15 @@ class Writer {
    */
   void WriteTag(Tag tag);
 
+  /**
+   * Writes a nanopb message to the output stream.
+   *
+   * This essentially wraps calls to nanopb's pb_encode() method. If we didn't
```

Comment:
Ultranit: since you're using backticks further on in the comment, consider wrapping pb_encode() in backticks as well.

Reply:
Done